homeassistant/components/tibber/sensor.py

```diff
@@ -66,11 +64,33 @@ def device_state_attributes(self):
         """Return the state attributes."""
         return self._device_state_attributes
 
+    @property
+    def model(self):
+        """Return the model of the sensor."""
+        return None
+
     @property
     def state(self):
         """Return the state of the device."""
         return self._state
 
+    @property
+    def device_id(self):
+        """Return the ID of the physical device this sensor is part of."""
+        return self.unique_id
```

Comment:
This is not correct. A device can have multiple entities, so each entity needs to have a unique ID that is device ID + what uniquely identifies the entity (like sensor type)